core-test/src/test/scala/org/bitcoins/core/psbt/PSBTSerializerTest.scala

```diff
@@ -0,0 +1,94 @@
+package org.bitcoins.core.psbt
+
+import org.bitcoins.testkit.util.BitcoinSUnitTest
+import scodec.bits._
+
+import scala.util.{Failure, Success, Try}
+
+/**
+  * Test vectors are taken directly from the BIP 174 reference sheet
+  * https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki#test-vectors
+  */
+class PSBTSerializerTest extends BitcoinSUnitTest {
+
+  val validPsbts: Vector[ByteVector] = Vector(
```

Comment:
I think it could be nice to assert the things we know should be in these PSBTs to ensure that deserialization happened correctly